airbyte-integrations/connectors/source-bigquery/unit_tests/integration/test_read_full_refresh.py

```diff
@@ -78,20 +125,18 @@ def test_basic(self, http_mocker: HttpMocker) -> None:
         http_mocker.post(
             BigqueryRequestBuilder.queries_endpoint(project_id=config["project_id"]).with_body(
                 build_query(
-                    project_id=config["project_id"],
                     dataset_id=dataset_id,
-                    table_id="INFORMATION_SCHEMA.KEY_COLUMN_USAGE",
-                    where=f"table_name='{table_id}'",
-                    query_end_char=";",
-                    timeout_ms=30000,
+                    table_id=table_id,
+                    dry_run=True,
+                    where="TRUE",
```

Comment:
I was very surprised with this where statement. Can we extract `"TRUE"` as a variable with a descriptive name like `_PUSH_FILTER_CONDITION`

Reply:
Sure.